lib/moose/test_case/reporter.rb

```diff
@@ -90,6 +92,10 @@ def passed_script
         message_with(:pass, "TEST Passed!")
       end
 
+      def skipped_script
+        message_with(:skipped, "TEST SKIPPED!")
```

Comment:
To follow the pattern from above lets make this `Skipped!` instead of being all caps